.eleventy.js

```diff
@@ -30,6 +30,33 @@ const DEV_MODE = process.env.ELEVENTY_RUN_MODE !== "build" // i.e. serve/watch
 module.exports = function(eleventyConfig) {
     eleventyConfig.setUseGitIgnore(false); // Otherwise docs are ignored
 
+    // Set DEV_MODE_POSTS to true if the context is not 'production'
+    const DEV_MODE_POSTS = process.env.CONTEXT !== "production";
+
+    let hasPrintedMessage = false;
+
+    // Define a filter named 'isFuturePost'
```

Comment:
Yeah, that's clear in the code :) 

```suggestion
    // The filter excludes blog posts with a date in the future for the production website.
```